docs/integrations/sources/notion.md

```diff
@@ -2,56 +2,69 @@
 :::
 
-2. Fill out the form. Make sure to check **Read content** and check any other [capabilities](https://developers.notion.com/reference/capabilities) you want to authorize.
-3. Click **Submit**.
-4. In the **Integration type** section, select either **Internal integration** (token authorization) or **Public integration** (OAuth2.0 authentication).
-5. Check the capabilities you want to authorize.
-6. If you select **Public integration**, fill out the fields in the **OAuth Domain & URIs** section.
-7. Click **Save changes**.
-8. Copy the Internal Access Token if you are using the [internal integration](https://developers.notion.com/docs/authorization#authorizing-internal-integrations), or copy the `access_token`, `client_id`, and `client_secret` if you are using the [public integration](https://developers.notion.com/docs/authorization#authorizing-public-integrations).
-
-### Step 2: Set up the Notion connector in Airbyte
-
-<!-- env:cloud -->
-**For Airbyte Cloud:**
-
-1. Log in to your [Airbyte Cloud](https://cloud.airbyte.com/workspaces) account.
-2. Click **Sources** and then click **+ New source**.
-3. On the Set up the source page, select **Notion** from the **Source type** dropdown.
-4. Enter a name for your source.
-5. Choose the method of authentication:
-      * If you select **Access Token**, paste the access token from [Step 8](#step-1-set-up-notion​).
-      * If you select **OAuth2.0** authorization, click **Authenticate your Notion account**.
-          * Log in and Authorize the Notion account. Select the permissions you want to allow Airbyte.
-6. Enter the **Start Date** in YYYY-MM-DDTHH:mm:ssZ format. All data generated after this date will be replicated. If this field is blank, Airbyte will replicate all data.
-7. Click **Set up source**.
-<!-- /env:cloud -->
-
-<!-- env:oss -->
-**For Airbyte Open Source:**
-
-1. Log in to your Airbyte Open Source account.
-2. Click **Sources** and then click **+ New source**.
-3. On the Set up the source page, select **Notion** from the **Source type** dropdown.
-4. Enter a name for your source.
-5. Choose the method of authentication:
-      * If you select **Access Token**, paste the access token from [Step 8](#step-1-set-up-notion​).
-      * If you select **OAuth2.0** authorization, paste the client ID, access token, and client secret from [Step 8](#step-1-set-up-notion​).
-6. Enter the **Start Date** in YYYY-MM-DDTHH:mm:ssZ format. All data generated after this date will be replicated. If this field is blank, Airbyte will replicate all data.
-7. Click **Set up source**.
-<!-- /env:oss -->
+2. Enter a **Name** for your integration. Make sure you have selected the workspace containing your data to replicate from the **Associated workspace** dropdown menu, and click **Submit**.
+3. In the navbar, select **Capabilities** and make sure to check **Read content**, as well as any other [capabilities](https://developers.notion.com/reference/capabilities) you want to authorize.
```

Comment:
what other capabilities should a user want to enable? should they also enable read comments and read user info ?